src/messages.rs

```diff
@@ -241,7 +241,7 @@ pub mod payload {
     use bytes::{Buf, BufMut, Bytes, BytesMut};
     use std::mem;
 
-    #[derive(Debug, Copy, Clone)]
```

Comment:
I removed this `Copy` because it was required only by the `Copy` trait of the `enum Payload`.
Since that it doesn't derive `Copy` anymore, I removed as unused

Furthermore I prefer explicit `Clone` instead of implicit `Copy` (to early detect any borrow-checker issues)